[Flaky] ./spec/system/admin/enterprises_spec.rb

### What we should change and why (this is tech debt)
<!-- The use of X changed and part y is now redundant. We can remove it and... -->
```
rspec ./spec/system/admin/enterprises_spec.rb:71 # 
    As an administrator
    I want to manage enterprises
 editing an existing enterprise
I, [2023-12-22T12:25:18.452106 #2849]  INFO -- : [knapsack_pro] To retry all the tests assigned to this CI node, please run the following command on your machine:
I, [2023-12-22T12:25:18.452168 #2849]  INFO -- : [knapsack_pro] bundle exec rspec --format progress --default-path spec "spec/system/admin/enterprises_spec.rb"
Stopped processing SimpleCov as a previous error not related to SimpleCov has been detected
Coverage report generated for RSpec to /home/runner/work/openfoodnetwork/openfoodnetwork/coverage. 6022 / 25254 LOC (23.85%) covered.
Error: Process completed with exit code 1.
```


### Context
<!-- This came up in PR #3451#issuecomment-461616636. -->
https://github.com/openfoodfoundation/openfoodnetwork/actions/runs/7299788184/job/19893269984


### Impact and timeline
<!-- Example: This simplifies the upgrade to Spree 3 and should be done after upgrading to Spree 2. -->

--- a/spec/system/admin/enterprises_spec.rb
+++ b/spec/system/admin/enterprises_spec.rb
@@ -101,6 +101,7 @@ describe '
     # expect(page).to have_checked_field "enterprise_enable_subscriptions_false"
 
     accept_alert do
+      scroll_to(:bottom)
       within(".side_menu") { click_link "Users" }
     end
     select2_select user.email, from: 'enterprise_owner_id'
@@ -116,14 +117,6 @@ describe '
     description_input.native.send_keys('This is an interesting long description')
 
     # Check StimulusJs switching of sidebar elements
-    accept_alert do
-      click_link "Primary Details"
-    end
-
-    # Back navigation loads the tab content
-    page.execute_script('window.history.back()')
-    expect(page).to have_selector '#enterprise_description'
-
     accept_alert do
       click_link "Primary Details"
     end
@@ -255,6 +248,14 @@ describe '
     )
     expect(page).to have_checked_field "enterprise_require_login_true"
     expect(page).to have_checked_field "enterprise_enable_subscriptions_true"
+
+    # Back navigation loads the tab content
+    page.execute_script('window.history.back()')
+    expect(page).to have_selector '#enterprise_description'
+
+    # Forward navigation brings back the previous tab
+    page.execute_script('window.history.forward()')
+    expect(page).to have_content 'This is my shopfront message.'
 
     # Test that the right input alert text is displayed
     accept_alert('Please enter a URL to insert') do

